validate.go

```diff
@@ -23,13 +24,14 @@ var (
 
 func init() {
 	var err error
-	firstPublishedCVE, err = time.Parse(TimeLayout, "2019-05-28T00:00Z")
+	firstPublishedCVE, err = time.Parse(types.TimeLayout, "2019-05-28T00:00Z")
 	if err != nil {
 		panic("Should not happen")
 	}
 }
 
-func validate(fileName string, vuln *Vuln) error {
+// Validate yaml files
+func Validate(fileName string, vuln *types.Vuln) error {
```

Comment:
`validate` instead of `Validate`. No need to make this accessible to other packages